src/Styles/book.css

```diff
@@ -0,0 +1,203 @@
+  align-items: center;
+  width: 1.5rem;
+}
+
+.Line-2 {
+  width: 0.125rem;
+  height: 0.5rem;
+  margin: 0.75rem 1.063rem 0 0.938rem;
+  border: solid 1px #e8e8e8;
+}
+
+.BookstoreBookEdit,
+.BookstoreBookRemove,
+.BookstroreComments {
+  width: 4.563rem;
+  height: 1.5rem;
+  margin: 0.188rem 0.938rem 0.125rem 0;
+  font-family: "RobotoSlab", serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  border-style: none;
+  background-color: #fff;
+  cursor: pointer;
+}
+
+.BookstoreBookLine {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.Oval-2 {
+  width: 4.25rem;
+  height: 4.25rem;
+  border-style: solid;
+  border-width: 5px;
+  border-image-source: linear-gradient(to bottom, #307bbe, var(--dodger-blue));
+  border-image-slice: 1;
+}
+
+.Rectangle-3 {
+  width: 5.625rem;
+  height: 5.62rem;
+  padding: 0.62rem 0.625rem 0.75rem 0.75rem;
+}
+
+.pourcentage {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.-Percent-Complete {
+  width: 4.25rem;
+  height: 2.438rem;
+  margin: 0.063rem 5.75rem 0 0.688rem;
+  font-family: "Montserrat", sans-serif;
+  font-size: 1.4rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.Completed {
+  width: 5rem;
+  height: 1.125rem;
+  margin: 0 4.938rem 0.75rem 0.75rem;
+  opacity: 0.5;
+  font-family: "Montserrat", sans-serif;
+  font-size: 0.875rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.BookstoreChaptres {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  gap: 0.5rem;
+}
+
+.Current-Chapter {
+  width: 10.875rem;
+  height: 1.125rem;
+  opacity: 0.5;
+  font-family: "RobotoSlab", serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+  text-transform: uppercase;
+}
+
+.Current-Lesson {
+  width: 4.688rem;
+  height: 1.313rem;
+  font-family: "RobotoSlab", serif;
+  font-size: 1rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.4px;
+  color: #121212;
+}
+
+.BookstoreUpdate {
+  width: 8.938rem;
+  height: 1.125rem;
+  font-family: "RobotoSlab", serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: #fff;
+  background-color: #0290ff;
+  border: none;
+}
+
+.Rectangle-2 {
+  width: 8.5rem;
+  height: 2.063rem;
+  padding: 0.438rem 1.188rem 0.5rem 1.375rem;
+  margin-right: 2.5rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+}
```

Comment:
- [ ] Please your Styling should be EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/)
### **Look at your design**
![yours](https://user-images.githubusercontent.com/55596970/193419690-ad72f436-2c8b-4924-9067-532ca91ded2a.png)
### **Zeplin Design**
![zeplin](https://user-images.githubusercontent.com/55596970/193419717-c284f6f0-9d07-405b-9b02-97f5decf656e.png)